Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -719,6 +719,24 @@ describe(
       );
     });
 
+    it("renders pickPrimitive during pick pass if defined", function () {
+      return Cesium3DTilesTester.loadTileset(scene, withBatchTableUrl).then(
+        function (batchTableTileset) {
+          var options = {
+            pickPrimitive: batchTableTileset,
+          };
+          return Cesium3DTilesTester.loadTileset(
+            scene,
+            tilesetUrl,
+            options
+          ).then(function (tileset) {
+            expect(tileset.pickPrimitive).toEqual(batchTableTileset);
+            expect(scene).toPickPrimitive(batchTableTileset);
```

Comment:
I copied the test over to `master` and noticed that this `expect` was passing. The default camera zoom might be too close, and I confirmed in a [Sandcastle example](http://localhost:8080/Apps/Sandcastle/index.html#c=tZJRT8IwFIX/SrOnkcyWiQjoICo8+GCiAaMve7lsF6h0LWm7ETX+d9sNCSKvLkt2b+85p1/TVaBJxXGLmgyJxC0Zo+FlQV/qtTANsrofK2mBS9Rp0LpOZeVcJkOJztS4ad26UTN0jUX9oOSS2zL3srOYduLBZf/6lwDsz7xNLwf9fu/iKMJNdkRj0NZVIDt0oVUxhdzVJjzaKTpK9rQ1Gs2gQA1UKLW+tTtbdHjke3SRcvnEbbaaglxi2KbtyIN3exGJu7Tdau3pLBdo0MaOr4nfaF5wyys0FPI8PMhtPp3Jc2MJP1NJSKnFFUkDSlnzzjaYGTYBC+yXnt2Bw8H85/vK7aoun2EukO0w6JtRMg1S+fUX8fzfEXeufXGSKoiCxNh3gSOf7Z8bXmyUtn6f0MVbLDYC3BWzeZmtnTszxh/GSxN2aE1yXhGeD0/8nCQTYIybLEohZvwD02CUMKf/YxWqvu3HCrWAdy9bxaOHZpFSmjDXnnZapcQc9FHyNw) that the camera was inside the larger building.

The fix should be pretty simple, at the beginning of the test add `viewRootOnly();`. Both tilesets will still be visible.